x-pack/plugins/stack_connectors/server/connector_types/bedrock/bedrock.test.ts

```diff
@@ -308,13 +403,101 @@ describe('BedrockConnector', () => {
           timeout: 120000,
           url: `${DEFAULT_BEDROCK_URL}/model/${DEFAULT_BEDROCK_MODEL}/invoke`,
           method: 'post',
-          responseSchema: RunActionResponseSchema,
+          responseSchema: RunApiLatestResponseSchema,
+          data: JSON.stringify({
+            anthropic_version: 'bedrock-2023-05-31',
+            system: 'Be a good chatbot',
+            messages: [
+              { content: 'Hello world', role: 'user' },
+              { content: 'Hi, I am a good chatbot', role: 'assistant' },
+              { content: 'What is 2+2?', role: 'user' },
+            ],
+            max_tokens: DEFAULT_TOKEN_LIMIT,
+            temperature: 0,
+          }),
+        });
+        expect(response.message).toEqual(mockResponseString);
+      });
+
+      it('adds system message from argument', async () => {
+        const response = await connector.invokeAI({
+          messages: [
+            {
+              role: 'user',
+              content: 'Hello world',
+            },
+            {
+              role: 'assistant',
+              content: 'Hi, I am a good chatbot',
+            },
+            {
+              role: 'user',
+              content: 'What is 2+2?',
+            },
+          ],
+          system: 'This is a system message',
+        });
+        expect(mockRequest).toBeCalledTimes(1);
+        expect(mockRequest).toHaveBeenCalledWith({
+          signed: true,
+          timeout: 120000,
+          url: `${DEFAULT_BEDROCK_URL}/model/${DEFAULT_BEDROCK_MODEL}/invoke`,
+          method: 'post',
+          responseSchema: RunApiLatestResponseSchema,
+          data: JSON.stringify({
+            anthropic_version: 'bedrock-2023-05-31',
+            system: 'This is a system message',
+            messages: [
+              { content: 'Hello world', role: 'user' },
+              { content: 'Hi, I am a good chatbot', role: 'assistant' },
+              { content: 'What is 2+2?', role: 'user' },
+            ],
+            max_tokens: DEFAULT_TOKEN_LIMIT,
+            temperature: 0,
+          }),
+        });
+        expect(response.message).toEqual(mockResponseString);
+      });
+
+      it('combines argument system message with conversation system message', async () => {
```

Comment:
@dgieselaar This test shows that we can accept a system argument and a system message. They will be combined.